nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server/src/main/java/org/apache/nifi/distributed/cache/server/DistributedSetCacheServer.java

```diff
@@ -35,6 +35,7 @@ protected CacheServer createCacheServer(final ConfigurationContext context) {
         final SSLContextService sslContextService = context.getProperty(SSL_CONTEXT_SERVICE).asControllerService(SSLContextService.class);
         final int maxSize = context.getProperty(MAX_CACHE_ENTRIES).asInteger();
         final String evictionPolicyName = context.getProperty(EVICTION_POLICY).getValue();
+        final int maxReadSize = context.getProperty(MAX_READ_SIZE).asInteger();
```

Comment:
Following the suggestion to use a Data Size Validator, this should be adjusted:
```suggestion
        final int maxReadSize = context.getProperty(MAX_READ_SIZE).asDataSize(DataUnit.B).intValue();
```

Reply:
If we go with the Data Size Validator, should `maxReadSize` be a `long`?  Anything over ~2.14 GB is over `Integer.MAX_SIZE`.  Or if we want to limit the max data size, perhaps additional custom validation is needed.